tests_integration/markets/seer/test_seer_agent_market.py

```
@@ -0,0 +1,39 @@
+from unittest.mock import Mock, patch
+
+from eth_pydantic_types import HexStr
+from web3 import Web3
+
+from prediction_market_agent_tooling.config import APIKeys
+from prediction_market_agent_tooling.gtypes import HexBytes, HexStr, CollateralToken
```

Ruff (0.11.9)

Redefinition of unused `HexStr` from line 3

(F811)